src/pages/experiments/[experimentId]/plots-and-tables/volcano/index.jsx

```diff
@@ -214,7 +230,11 @@ const VolcanoPlot = () => {
           description='Could not load differential expression data.'
           onClick={() => {
             dispatch(
-              loadDifferentialExpression(experimentId, config.diffExpData),
+              loadDifferentialExpression(
```

Comment:
great that you are fixing the volcano plot as well. Can you also remove the outdated cell set options from the initial state for the volcano plot? They are wrongly placed in the styling config, which is bad. This is the line that needs removing: https://github.com/biomage-ltd/ui/blob/master/src/redux/reducers/componentConfig/initialState.js#L88 Make sure that nothing reads from it, that the new part of the redux store that you are implementing is taken into an account only.

Reply:
Removed cellSets in initial config, checked no proble with the plot. However, because there is no cell selected, the plot's initial state is loading. I think this can be released as a new ticket.

Reply:
can you make a new ticket for that, Agi? Please send me a link to it once you make it.